docs/source/schemas/organization.md

```diff
@@ -0,0 +1,354 @@
+---
+title: Organization
+description: Scaling your Apollo Server from a single file to your entire team
+---
+
+## Prerequisites
+
+* Understanding of GraphQL types
+* Resolvers
+
+## Overview
+
+The schema contains the information to define all requests that the client can request from an instance of Apollo Server along with the resolvers necessary to route the requests to retrieve data. For most applications, the schema type definitions are be placed in a single file along side the resolvers. Placing reolvers in the same file as their accompanying type definitions is the best way to organize the code, since it enables developers to locate and modify the two inter-dependent portions.
```

Comment:
>  For most applications, the schema type definitions are be placed in a single file along side the resolvers.

"the schema type definitions are be placed" => "the schema type definitions are placed"...